app/scenes/channel_members/channel_members.js

```diff
@@ -19,29 +20,25 @@ export default class ChannelMembers extends PureComponent {
     static propTypes = {
         currentChannel: PropTypes.object,
         currentChannelMembers: PropTypes.array.isRequired,
+        currentChannelMemberCount: PropTypes.number.isRequired,
         currentTeam: PropTypes.object,
         preferences: PropTypes.object,
+        requestStatus: PropTypes.string,
         actions: PropTypes.shape({
             getProfilesInChannel: PropTypes.func.isRequired
         })
     }
 
-    state = {
-        currentChannelMemberCount: 0
-    }
-
     componentDidMount() {
-        this.props.actions.getProfilesInChannel(this.props.currentTeam.id, this.props.currentChannel.id, 0);
-    }
-
-    componentWillReceiveProps(nextProps) {
-        this.setState({
-            currentChannelMemberCount: this.state.currentChannelMemberCount + nextProps.currentChannelMembers.length
+        InteractionManager.runAfterInteractions(() => {
+            this.props.actions.getProfilesInChannel(this.props.currentTeam.id, this.props.currentChannel.id, 0, 200);
```

Comment:
200 profiles at a time? we have 100 in the webapp, this one concerns me a little bit